src/incubator/expandableOverlay/index.tsx

```diff
@@ -1,16 +1,18 @@
-import React, {useCallback, useState, forwardRef, PropsWithChildren, useImperativeHandle} from 'react';
+import React, {useCallback, useState, forwardRef, PropsWithChildren, useImperativeHandle, useRef} from 'react';
 
 import TouchableOpacity, {TouchableOpacityProps} from '../../components/touchableOpacity';
 import View from '../../components/view';
 import Modal, {ModalProps, ModalTopBarProps} from '../../components/modal';
 import DialogOld from '../../components/dialog';
 import DialogNew, {DialogMigrationProps} from '../dialog';
 import {Colors} from 'style';
+import {AccessibilityInfo, findNodeHandle} from 'react-native';
```

Comment:
We usually put more general imports higher up; for example I'd put `react-native` below `react`